src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,8 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    public boolean isMutating() {
```

Comment:
Since this is the first time this method created, you should have a header comment here to explain.
One thing is worth to note is that even though inheritance allows you to reduce the amount of code, does it truly reflect the intuitiveness and natural function of **isMutating** method? Is there any way to make each subclass have to redefine on its own this same method?